integration-tests/mllp/src/test/java/org/apache/camel/quarkus/component/mllp/it/MllpTest.java

```diff
@@ -16,8 +16,6 @@
  */
 package org.apache.camel.quarkus.component.mllp.it;
 
-import io.quarkus.test.common.QuarkusTestResource;
```

Comment:
I don't think removing the `QuarkusTestResource` import was part of the original commit. Pretty sure removing it will result in a compilation error.